docs/user-guide/creating-application/workflow/cd-pipeline.md

```diff
@@ -84,7 +84,20 @@ Make sure your cluster has `devtron-agent` installed if you check the `Execute i
 
 ![](../../../.gitbook/assets/cd_pre_build%20%282%29.jpg)
```

Comment:
Can we put this in s3 instead of our repo?

Reply:
This doesn't seems resolved. @ashokdevtron

Reply:
Now it's done, @prakarsh-dt